embed.html

```diff
@@ -82,19 +82,6 @@
     }
 
 
-//--></script> 
-
-<!-- Piwik -->  
-<script type="text/javascript"> 
-var pkBaseURL = (("https:" == document.location.protocol) ? "https://openstreetmap.by/piwik/" : "http://openstreetmap.by/piwik/"); 
-document.write(unescape("%3Cscript src='" + pkBaseURL + "piwik.js' type='text/javascript'%3E%3C/script%3E")); 
-</script><script type="text/javascript"> 
-try { 
-var piwikTracker = Piwik.getTracker(pkBaseURL + "piwik.php", 1); 
-piwikTracker.trackPageView(); 
-piwikTracker.enableLinkTracking(); 
-} catch( err ) {} 
-</script><noscript><p><img src="http://openstreetmap.by/piwik/piwik.php?idsite=1" style="border:0" alt="" /></p></noscript> 
-<!-- End Piwik Tracking Code --> 
+//--></script>
```

Comment:
Are there any site stats over these past years left? Using self-hosted analytics is a good way of protecting users.

Reply:
not sure, @Komzpa can you help with answering there, this code doesn't work on http://openstreetmap.by/embed.html, see console error

Reply:
Opened #15 to track this. Can you see the traces of `piwik` database on the server? https://matomo.org/faq/how-to-install/faq_138/